Package.swift

```diff
@@ -512,7 +512,9 @@ let package = Package(
         ),
         .package(url: "https://github.com/tuist/Path", .upToNextMajor(from: "0.3.0")),
         .package(
-            url: "https://github.com/tuist/XcodeGraph.git", exact: "1.3.2"
+            /// I temporarily set it to my repository and submitted a PR with the extendingFile case
+            /// added to XcodeGraph’s InfoPlist.
+            url: "https://github.com/sanghyeok-kim/XcodeGraph.git", branch: "add-InfoPlist-case-extendingFile"
```

Comment:
This should be changed back to the original repository URL and the version with the extendingFile merged.
- https://github.com/tuist/XcodeGraph/pull/97